File: Computabilidade-e-Complexidade-de-Algoritmos-main/golang/23.go
package main
import (
    "fmt"
    "strings"
)

func contem101Ou110(s string) bool {
    return strings.Contains(s, "101") || strings.Contains(s, "110")
}

func main() {
    fmt.Println(contem101Ou110("1101"))  // True
    fmt.Println(contem101Ou110("1001"))  // False
}

File: Computabilidade-e-Complexidade-de-Algoritmos-main/java/23.java
public class AFNContem101Ou110 {
    public static boolean contem101Ou110(String s) {
        return s.contains("101") || s.contains("110");
    }

    public static void main(String[] args) {
        System.out.println(contem101Ou110("1101"));  // True
        System.out.println(contem101Ou110("1001"));  // False
    }
}
